src/h3lib/include/baseCells.h

```diff
@@ -46,5 +46,8 @@ extern const BaseCellData baseCellData[NUM_BASE_CELLS];
 int _isBaseCellPentagon(int baseCell);
 int _faceIjkToBaseCell(const FaceIJK* h);
 int _faceIjkToBaseCellCCWrot60(const FaceIJK* h);
+void _baseCellToFaceIjk(int baseCell, FaceIJK* h);
+bool _baseCellIsCwOffset(int origin, int test);
+int _getBaseCellNeighbor(int baseCell, int dir);
```

Comment:
I thought the convention we were using was that `_`-prefixed functions were local to the module? We never went through and did a clean sweep here, but it might be worthwhile.

Reply:
We should also add some developer documentation that explains the difference between H3_EXPORT, unprefixed, and prefixed functions.